#ERROR: unknown scenario 'RUN'
# press CTRL+F5 to run this scenario with current py/repo (under vscode)

RUN:
      
  - GET: http://test/test
    tests:
      - R.status == 200

  - CALL: RUN     # <= can't CALL itself